test/integration/unlogged_in_registered_user_must_login_to_checkout_test.rb

```diff
@@ -14,22 +14,22 @@ class UnloggedInRegisteredUserMustLoginToCheckoutTest < ActionDispatch::Integrat
 
     visit "/cart"
     click_button "Checkout"
-    
+
     assert "/login", current_path
 
     assert page.has_content?("Must Login")
-     fill_in 'Username', with: 'ryan'
-     fill_in "Password", with: 'waffles'
-     click_button "Login"
+    fill_in 'Username', with: 'ryan'
+    fill_in "Password", with: 'waffles'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.